packages/components/src/mobile/bottom-sheet/cell.native.js

```diff
@@ -192,6 +194,29 @@ class BottomSheetCell extends Component {
 			}
 		};
 
+		const setTextInputValue = ( text ) => {
+			this.setState( { localValue: text } );
+			this.props.onChangeValue( text );
+		};
+
+		const clearButton = () => {
+			return (
+				<>
```

Comment:
I'm wondering if we need to use a fragment here since we're only returning a single instance 🤔 .

Reply:
I think my reasoning here was that in previous projects, the code styles had dictated that you could not return a conditional rendering block using `{ ... }` without using a Fragment as the parent of the curly braced block, so perhaps this was habit. This has been removed, however.

Reply:
Oh, that's interesting, I was also accustomed to doing something similar in these cases to prevent returning a `false` value, in case the condition isn't fulfilled, instead of an instance, which might produce render issues on Android. I see in the recent changes that this is no longer an issue, so I'm marking this comment as resolved 👍 .